.travis.yml

```diff
@@ -32,6 +32,7 @@ install: "echo no install step required"
 
 script:
   - go test ./...
```

Comment:
if we're running tests with `-race`, is it really worth running them without too?

Reply:
I don't know for sure, but per Slack chat let's drop the non-race line and we can read up for the guarantees or otherwise of `-race`